lib/routes.test.js

```diff
@@ -27,6 +27,7 @@ describe(_.startCase(filename), function () {
     fakeLog = sandbox.stub();
     fakeAuth = sandbox.stub();
     fakeAuth.addRoutes = sandbox.stub();
+    fakeAuth.useAuth = sandbox.stub();
```

Comment:
Are we asserting that `useAuth` is being called? If not, then there should be a test case which makes the assertion to make sure the method is being called with the expected arguments